packages/synapse-react-client/src/components/GenericCard/GenericCard.tsx

```diff
@@ -45,6 +45,7 @@ import { useQueryContext } from '../QueryContext'
 import { convertDoiToLink } from '../../utils/functions/RegularExpressions'
```

Comment:
if it's not too hard (I think the current GenericCard test setup might be kind of a mess), adding a test to GenericCard to make sure the button appears when it should would be great too